dm1_registers.xml

```diff
@@ -15,7 +15,14 @@
             system. To perform a reset the debugger writes 1, and then writes 0
             to deassert the reset.
         </field>
-        <field name="0" bits="29:26" access="R" reset="0" />
+        <field name="dmactive" bits="29" access="R/W" reset="0">
+          This bit indicates whether the Debug Module is active. When 0,
+	  all DM registers and state keep their reset values.
+	  DM registers should not be otherwise reset after power-on,
+	  neither by the platform's system reset or Debug Transport
+	  reset signals.
```

Comment:
Sorry about the whitespaces... you merged the last one before I fixed it... will see what's up with my editor. No need to merge this yet.